docs/wiki/DeploymentFlow.md

```diff
@@ -30,6 +30,17 @@ Modules in this reference implementation must be deployed in the following order
 |   8   | Built-In and Custom Policy Assignments | Creates policy assignments to provide governance at scale.                                                                                                                                 | Management Groups, Log Analytics Workspace & Custom Policy Definitions | [infra-as-code/bicep/modules/policy/assignments/alzDefaults](https://github.com/Azure/ALZ-Bicep/tree/main/infra-as-code/bicep/modules/policy/assignments/alzDefaults) |
 |   9   | Corp Connected Spoke Network           | Creates Spoke networking infrastructure with Virtual Network Peering to support Hub & Spoke network topology.  Spoke subscriptions are used for deploying construction sets and workloads. | Management Groups, Hub Networking & Subscription for spoke networking  | [infra-as-code/bicep/modules/spokeNetworking](https://github.com/Azure/ALZ-Bicep/tree/main/infra-as-code/bicep/modules/spokeNetworking)                               |
 
+## Other Deployment Scenarios
+
+The following modules can be used to deploy components representing different scenarios according to the Azure Landing Zone conceptual architecture.
+
+### Scenario: On-premises connectivity using Azure vWAN
```

Comment:
Do we need to add some text here to say you still need to do steps 1,2,3 etc (the ones that are applicable)

Reply:
@jtracey93 as discussed, I'm creating sub markdowns for network topology deployment of H&S and VWAN. Also updated the high-level deployment flow to include 7 (instead of 9) steps.
https://github.com/Azure/ALZ-Bicep/pull/152/commits/22aab293928ca0a14ed5c2d8e41e18bb4025a63e